Add resolve conflicts option on addon create (#5880)
weaveworks/eksctl: userdocs/src/usage/addons.md

eksctl create addon --name vpc-cni --version 1.7.5 --service-account-role-arn=<role-arn>
```

> During addon creation, providing a self-managed version of the addon already exists on the cluster, you have the possibility to choose how potential `configMap` conflicts shall be resolved. This is achievable via the config file field `resolveConflicts`. e.g.,

Reviewer: ```suggestion
During addon creation, if a self-managed version of the addon already exists on the cluster, you can choose how potential `configMap` conflicts shall be resolved by setting `resolveConflicts` option via the config file. e.g.,
```

Author: https://github.com/weaveworks/eksctl/pull/5880/commits/23ba75f905f9d82453af4d9228c29fc5eda7d7a9
